eldritch/events.py

```diff
@@ -1511,6 +1533,8 @@ def resolve(self, state):
       return False
     if self.check.successes >= self.monster.toughness(state):
       # TODO: take the monster as a trophy
+      # Stand-in until we implement trophy code to allow MoveMultipleThroughMonster test to work
+      self.monster.place = None
```

Comment:
You set up some trophy code above, inside of the Combat event?

Reply:
Yes, though that code is wrong too. Probably TakeAsTrophy (or similar) should be an event, so we can hijack it for things like Loot Piñatas, personal stories, etc. As a wise man once said, "Passing information between events is most of what happens in this game."

Probably there should only be one bit where I move the monster, though. Any preferences on it appearing in Combat or CombatRound?

Reply:
I think either Combat or CombatRound works. I can't think of a reason why one would work better than the other.

Reply:
So I realize I said last time that you could do this in either Combat or CombatRound. And I also realize this is a placeholder. But at a minimum, the code for taking the trophy (adding it the player's possessions) and the code for setting the monster's place to None should live in the same place - otherwise someone is going to be very confused in the future.